src/wp-login.php

```diff
@@ -180,9 +180,27 @@ function login_header( $title = 'Log In', $message = '', $wp_error = null ) {
 	 * @since WP-4.6.0
 	 */
 	do_action( 'login_header' );
+
+	/**
+	* Get the image for login page from the theme logo via the customizer
+	*
+	* @since 1.0.2
+	*/
+	$default_logo_img_url = admin_url('/images/w-logo-blue.png?ver=20190218');
```

Comment:
This function call needs spaces, and this image is only 80x80px, being displayed in a space that is 84x84px.

How about we use use the SVG image instead?

```php
$default_logo_img_url = admin_url( '/images/wordpress-logo.svg?ver=20190218' );
```

@bahiirwa you are probably a good person to ask about browser compatibility here, do you ever come across devices that are still running IE 8 or lower?  Otherwise, I think this is OK, we have already used SVG images elsewhere in the admin dashboard.  Reference: https://caniuse.com/#feat=svg-img

Reply:
By default, this particular point of CP/WP takes images about 80px tall and wide. 

Will fix the space.

As for devices, not really. IE8 is the default browser for Windows 7 and Windows Server 2008 R2 operating systems however, coming from a continent that is not so forward in tech devices, most windows users run Windows 7 but with other modern browsers like mozilla and chrome. 

The pixels disparity might not be a big issue with 4x4px difference. However, we could add a bigger png by default and avoid the guess work. Or take the big leap and go for SVG.

Reply:
On the other hand have pushed a css change with `width:80px` and `height:80px`

Reply:
A bigger issue than `80px` vs `84px` is that `80px` is not really 80 pixels on modern devices with high screen density, and each year there are more of these with higher pixel density.  Let's go with the SVG here.

Reply:
Change effected `$default_logo_img_url = admin_url( '/images/wordpress-logo.svg?ver=20190218' );` for hi-res devices.

Could we rename the image to classicpress-logo.svg? Will this have wide ranging issues?
I see some WP Branding infringement is seen here.

Reply:
Any comment on the branding rename?